src/api.ts

```diff
@@ -1,7 +1,7 @@
 import { MovieData } from './types/MovieData';
 import { ResponseError } from './types/ReponseError';
 
-const API_URL = 'https://www.omdbapi.com/?apikey=your-key';
+const API_URL = 'http://www.omdbapi.com/?i=tt3896198&apikey=81ae11e5';
```

Comment:
```suggestion
const API_URL = 'https://www.omdbapi.com/?i=tt3896198&apikey=81ae11e5';
```
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/55802630/187864617-9f549202-9a2f-4747-906c-1c54a2e34ffa.png">